bgpd/bgp_packet.c

```diff
@@ -2019,8 +2019,10 @@ static int bgp_capability_msg_parse(struct peer *peer, u_char *pnt,
 
 			/* Ignore capability when override-capability is set. */
 			if (CHECK_FLAG(peer->flags,
-				       PEER_FLAG_OVERRIDE_CAPABILITY))
+				       PEER_FLAG_OVERRIDE_CAPABILITY)) {
+				pnt += hdr->length + 3;
```

Comment:
nit: maybe we could increment `pnt` right after the memcpy so we need to do it only once